Simple emscripten wasm causes error

```
(module
  (type $t0 (func (result i32)))
  (type $t1 (func (param i32 i32 i32) (result i32)))
  (import "env" "_emscripten_memcpy_big" (func $env._emscripten_memcpy_big (type $t1)))
  (func $main (export "main") (type $t0) (result i32)
    i32.const 0)
  (memory $memory (export "memory") 2 10)
  (global $g0 i32 (i32.const 0))
  (global $g1 (mut i32) (i32.const 0)))
```

```
wasmer run wasmer.wasm
thread 'main' panicked at 'index out of bounds: the len is 0 but the index is 0', /home/circleci/project/lib/runtime-core/src/structures/slice.rs:58:10
note: Run with `RUST_BACKTRACE=1` for a backtrace.
Makefile:2: recipe for target 'run' failed
make: *** [run] Error 101
```

--- a/lib/emscripten/src/utils.rs
+++ b/lib/emscripten/src/utils.rs
@@ -34,11 +34,13 @@ pub fn is_emscripten_module(module: &Module) -> bool {
 }
 
 pub fn get_emscripten_table_size(module: &Module) -> (u32, Option<u32>) {
+    assert!(module.info().imported_tables.len() > 0, "Emscripten requires at least one imported table" );
     let (_, table) = &module.info().imported_tables[ImportedTableIndex::new(0)];
     (table.minimum, table.maximum)
 }
 
 pub fn get_emscripten_memory_size(module: &Module) -> (Pages, Option<Pages>, bool) {
+    assert!(module.info().imported_tables.len() > 0, "Emscripten requires at least one imported memory" );
     let (_, memory) = &module.info().imported_memories[ImportedMemoryIndex::new(0)];
     (memory.minimum, memory.maximum, memory.shared)
 }

--- a/lib/emscripten/src/utils.rs
+++ b/lib/emscripten/src/utils.rs
@@ -34,13 +34,19 @@ pub fn is_emscripten_module(module: &Module) -> bool {
 }
 
 pub fn get_emscripten_table_size(module: &Module) -> (u32, Option<u32>) {
-    assert!(module.info().imported_tables.len() > 0, "Emscripten requires at least one imported table" );
+    assert!(
+        module.info().imported_tables.len() > 0,
+        "Emscripten requires at least one imported table"
+    );
     let (_, table) = &module.info().imported_tables[ImportedTableIndex::new(0)];
     (table.minimum, table.maximum)
 }
 
 pub fn get_emscripten_memory_size(module: &Module) -> (Pages, Option<Pages>, bool) {
-    assert!(module.info().imported_tables.len() > 0, "Emscripten requires at least one imported memory" );
+    assert!(
+        module.info().imported_tables.len() > 0,
+        "Emscripten requires at least one imported memory"
+    );
     let (_, memory) = &module.info().imported_memories[ImportedMemoryIndex::new(0)];
     (memory.minimum, memory.maximum, memory.shared)
 }

